dist/tools/backport_pr/backport_pr.py

```diff
@@ -135,6 +135,13 @@ def main():
     if status != 200:
         print("Could not retrieve user: {}".format(user['message']))
         exit(1)
+    scopes = g.getheaders()
+    scopes = dict(scopes)["X-OAuth-Scopes"]
+    scopes = scopes.split(',')
+    scopes = [x.strip() for x in scopes]
+    if not('public_repo' in scopes or 'repo' in scopes):
```

Comment:
`not` is not a function, so
```suggestion
    if not ('public_repo' in scopes or 'repo' in scopes):
```

Reply:
I will add the space after not.
I will leave the split as it is since there is no import re yet , an i like easier to read.